### PR TITLE
[RHBA-685-7.7.x]: Redirected to Spaces when Organizational Unit is not active

### PR DESCRIPTION
**Pull request by adrielparedes**
If an user has no rights to create a Space, the default project could not be created. So the exception was catch and redirected to SpacesScreen to select another Space where the user has rights to see and edit.

Related to:
https://github.com/kiegroup/appformer/pull/328
https://github.com/kiegroup/kie-wb-common/pull/1715

### CODE REVIEW

File: kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/util/LibraryPlaces.java [hunk 1]
```diff
@@ -564,8 +565,11 @@ public void callback(OrganizationalUnit organizationalUnit) {
                             projectContextChangeEvent.fire(new WorkspaceProjectContextChangeEvent(organizationalUnit));
                             setupLibraryPerspective();
                         }
-                    }
-            ).getDefaultOrganizationalUnit();
+                    },
+                    (message, throwable) -> {
+                        this.goToOrganizationalUnits();
```
Comment (mbarkley) [inline kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/util/LibraryPlaces.java]: If you use an UnauthorizedException (from Errai -- see comment on the appformer PR), you could throw and catch it here, and handle that case separately. For example:

    (message, throwable) -> {
      try {
        throw throwable;
      } catch (UnauthorizedException ue) {
        this.goToOrganizationalUntis();
        return false;
      } catch (Throwable t) {
        return true; // Let default error handling happen.
      }
    }

**Comment by adrielparedes** [comment]
@mbarkley thanks for the suggestion, I changed it

**Comment by cristianonicolai** [comment]
@adrielparedes could you include a basic test to cover your chages?

**Comment by jhrcek** [comment]
Jenkins execute full downstream build

**Comment by adrielparedes** [comment]
@cristianonicolai I've added the missing test. I have to modify something here: https://github.com/kiegroup/appformer/pull/328 Would you mind to review it please?

**Comment by cristianonicolai** [comment]
jenkins please retest this

**Pull request merged**